app/services/profiles/extract_data.rb

```diff
@@ -0,0 +1,47 @@
+module Profiles
+  module ExtractData
+    DIRECT_ATTRIBUTES = %i[
```

Comment:
Note that this hard-coded for DEV values. We'll keep this in prod for a bit until all profiles have been migrated, then generalize this.